docs/models/icon/usage.md

```diff
@@ -4,23 +4,41 @@
 
 The [ICON repository :material-open-in-new:](https://github.com/C2SM/icon){:target="_blank"} is hosted on the C2SM GitHub organisation. If you do not have access, please follow the instructions under [How to get Access](../../about/index.md#how-to-get-access).
 
-Once you have access, clone the repository from GitHub using the SSH protocol:
-
-  ```bash
-  git clone --recurse-submodules git@github.com:C2SM/icon.git
-  ```
   If you do not already have an SSH key set up for GitHub, but would like to do so, follow the [instructions :material-open-in-new:](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent){:target="_blank"}.
     
 ## Configure and compile
 
 ### Säntis
 
-!!! construction "Under construction - last update: 2025-02-09"
+!!! construction "Under construction - last update: 2025-02-18"
 
     Information on this section is not yet complete nor final. It will be updated following the progress of the Alps system deployment at CSCS and C2SM's adaptation to this new system. Please use the [C2SM support forum :material-open-in-new:](https://github.com/C2SM/Tasks-Support/discussions){:target="_blank"} in case of questions regarding building ICON on Alps.
 
-Next, follow the instructions to build ICON using Spack below.
+#### ICON at C2SM
+
+Clone the ICON repository on the branch `santis`:
+```console
+git clone -b santis --recurse-submodules git@github.com:C2SM/icon.git
+```
+
+Run the following after navigating into ICON root folder:
+```console
+# Load ICON user-environment 
+CLUSTER_NAME=todi uenv start --view=spack icon-wcp/v1:rc4
+SPACK_TAG=$(cat "config/cscs/SPACK_TAG_ALPS")
+
+# Setup spack
+git clone --depth 1 --recurse-submodules --shallow-submodules -b ${SPACK_TAG} https://github.com/C2SM/spack-c2sm.git
+. spack-c2sm/setup-env.sh /user-environment
+
+# Build ICON
+cd /path/to/icon-build-folder
```

Comment:
I would remove this and add `&& cd icon` to the first code block (when ICON is cloned).

Because right afterwards one assumes that you are already in your icon folder (`SPACK_TAG=...`).

Same above.

Reply:
the icon-build-folder doesn't have to be the same as the icon root folder. That's for out-of-source builds we recommend in large_use_cases

Reply:
But here there is no mention of out-of-source. Also, you can't copy the whole block if there is a `/path/to/icon-build-folder`.

Reply:
better: https://github.com/C2SM/c2sm.github.io/pull/252/commits/00963f6c9bf23fdd3811b7e5fb12db7c41f892a0?

Reply:
Almost! I like the comment regarding out-of-source, but would still include `cd icon` / `cd icon-nwp`.